docs/content/en/docs/getting-started/production-environment/cloudstack-getstarted.md

```diff
@@ -3,6 +3,226 @@ title: "Create CloudStack production cluster"
+   ```bash
+   export EKSA_AWS_ACCESS_KEY_ID="your*access*id"
+   export EKSA_AWS_SECRET_ACCESS_KEY="your*secret*key"  
+   ```
+     
+1. Disable Kubevip load balancer
+
+   Skip this step if you want to use the Kubevip load balancer with your cluster. If you want to use a different load balancer, you can disable Kubevip as follows:
+
+   ```bash
+   export CLOUDSTACK_KUBE_VIP_DISABLED=true
+   ```
+1. Create cluster
+
+   ```bash
+   eksctl anywhere create cluster -f eksa-mgmt-cluster.yaml
+   ```
+
+1. Once the cluster is created you can use it with the generated `KUBECONFIG` file in your local directory:
+
+   ```bash
+   export KUBECONFIG=${PWD}/${CLUSTER_NAME}/${CLUSTER_NAME}-eks-a-cluster.kubeconfig
+   ```
+1. Check the cluster nodes:
+
+   To check that the cluster completed, list the machines to see the control plane, etcd, and worker nodes:
+
+   ```bash
+   kubectl get machines -A
+   ```
+
+   Example command output
+   ```
+   NAMESPACE   NAME                PROVIDERID        PHASE    VERSION
+   eksa-system mgmt-b2xyz          cloudstack:/xxxxx    Running  v1.23.1-eks-1-21-5
+   eksa-system mgmt-etcd-r9b42     cloudstack:/xxxxx    Running  
+   eksa-system mgmt-md-8-6xr-rnr   cloudstack:/xxxxx    Running  v1.23.1-eks-1-21-5
+   ...
+   ```
+
+   The etcd machine doesn't show the Kubernetes version because it doesn't run the kubelet service.
+
+1. Check the initial cluster's CRD:
+
+   To ensure you are looking at the initial cluster, list the CRD to see that the name of its management cluster is itself:
+
+   ```bash
+   kubectl get clusters mgmt -o yaml
+   ```
+
+   Example command output
+   ```
+   ...
+   kubernetesVersion: "1.23"
+   managementCluster:
+     name: mgmt
+   workerNodeGroupConfigurations:
+   ...
+   ```
+
+   {{% alert title="Note" color="primary" %}}
+   The initial cluster is now ready to deploy workload clusters.
+   However, if you just want to use it to run workloads, you can deploy pod workloads directly on the initial cluster without deploying a separate workload cluster and skip the section on running separate workload clusters.
+   To make sure the cluster is ready to run workloads, run the test application in the [Deploy test workload section.]({{< relref "../../tasks/workload/test-app" >}})
+   {{% /alert %}}
+
+### Create separate workload clusters
+
+Follow these steps if you want to use your initial cluster to create and manage separate workload clusters.
+
+1. Generate a workload cluster config:
+   ```bash
+   CLUSTER_NAME=w01
+   eksctl anywhere generate clusterconfig $CLUSTER_NAME \
+      --provider cloudstack > eksa-w01-cluster.yaml
+   ```
+
+   Refer to the initial config described earlier for the required and optional settings.
+   Ensure workload cluster object names (`Cluster`, `CloudDatacenterConfig`, `CloudStackMachineConfig`, etc.) are distinct from management cluster object names.
```

Comment:
Should we here say, to make sure that the managementCluster field on the cluster spec is updated to the right cluster name?

Reply:
Great point! I'll do that.

Reply:
Done.